source/docs/zero-to-robot/step-4/creating-benchtop-test-program-cpp-java.rst

```diff
@@ -3,18 +3,20 @@ Creating your Benchtop Test Program (C++/Java)
 
 Once everything is installed, we're ready to create a robot program.  WPILib comes with several templates for robot programs.  Use of these templates is highly recommended for new users; however, advanced users are free to write their own robot code from scratch. This article walks through creating a project from one of the provided examples which has some code already written to drive a basic robot.
 
+.. important:: This guide includes code examples that involve vendor hardware at the convenience of the user. In this document, PWM refers to the motor controller included in the KOP. The CTRE tab references the Falcon 500 motor controller, but is applicable on the TalonSRX and VictorSPX as well. REV examples will be coming soon.
```

Comment:
There should be a reference to installing vendor software and links to the relevant documentation.